### PR TITLE
Add read_timeout parameter to get() and send()

### PR DESCRIPTION
**Pull request by dagwieers**
This adds a read_timeout parameter to influence the read timeout value separately from the connection timeout value.

### CODE REVIEW

File: pypsrp/wsman.py [hunk 1]
```diff
@@ -718,7 +721,7 @@ def send(self, message):
         return self._send_request(prep_request, hostname)
 
     def _send_request(self, request, hostname):
-        response = self.session.send(request, timeout=self.connection_timeout)
+        response = self.session.send(request, timeout=(self.connection_timeout, self.read_timeout))
```
[the remark this answers is not in the document]
Comment (nitzmahone) [reply pypsrp/wsman.py]: I went and looked at the code for 2.9.1 and the tuple support is there

[the remark this answers is not in the document]
Comment (dagwieers) [reply pypsrp/wsman.py]: Thanks, so can we merge this as-is ?

File: pypsrp/wsman.py [hunk 2]
```diff
@@ -310,7 +311,7 @@ def signal(self, resource_uri, resource, option_set=None,
     def get_server_config(self, uri="config"):
         resource_uri = "http://schemas.microsoft.com/wbem/wsman/1/%s" % uri
         log.info("Getting server config with URI %s" % resource_uri)
-        return self.get(resource_uri)
+        return self.get(resource_uri, timeout=(self.connection_timeout, self.read_timeout))
```
[the remark this answers is not in the document]
Comment (dagwieers) [reply pypsrp/wsman.py]: Removed.

**Comment by dagwieers** [comment]
Please re-review.

**Comment by dagwieers** [comment]
Please re-review.

**Comment by jborean93** [comment]
Looking good, I'll update the changelog and add the feature flags in a separate PR once I think of the way forward for those things.

**Pull request merged**